### PR TITLE
Help Page Update 5

### PR DESCRIPTION
**Pull request by antares1470**
### Description of the Change

- updated all the help pages in the File Operations section (merged some pages together)
- updated all the help pages in the Visualisation section, moved Perspective Bookmarks to Views section
- updated all the help pages in the Selection section (merged some pages together), added some new pages as well
- updated all the analytic help pages
- updated some of the existing arrangement help pages and added a couple of new ones, removed help pages for Bubble Tree (temporarily until we add some proper help for it), Grid (General), and Clusters on Hilbert Curve

### Why Should This Be In Core?

Updated help

### Benefits

More up to date help

### Possible Drawbacks

Nil

### Applicable Issues

#340

### CODE REVIEW

**Review by aldebaran30701 — CHANGES_REQUESTED** [review]
Very thorough updates - Just some minor points but completely happy with this!

**Review by Delphinus8821 — APPROVED** [review]

File: CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/docs/flatten-z-field.html [hunk 1]
```diff
@@ -1,16 +1,39 @@
...
+            </tr>
+            <tr>
+                <td>
+                    Run Flatten Z-Field
+                </td>
+                <td>
+                    
+                </td>
+                <td>
+                    Arrange -&gt; Flatten Z Field
```
[the remark this answers is not in the document]
Comment (antares1470) [reply CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/docs/flatten-z-field.html:26]: Ah I thought I got all those. Will make that change

File: CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/docs/flatten-z-field.html [hunk 2]
```diff
@@ -1,16 +1,39 @@
...
+                </td>
+            </tr>
+        </table>
+        
         <p>
-            Set the Z Attributes for all the nodes to zero.
-            <br>The operation is performed on all nodes, regardless of the current selection set.
-            <br>May be used to transform a graph from 3D to 2D mode.
+            Flatten Z-Field sets the Z attribute of all nodes to zero. This is performed on all nodes, 
+            regardless of the current selection.
```
[the remark this answers is not in the document]
Comment (aldebaran30701) [reply CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/docs/flatten-z-field.html:36]: @antares1470 Just wondering your thoughts about this.

[the remark this answers is not in the document]
Comment (antares1470) [reply CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/docs/flatten-z-field.html:36]: @aldebaran30701 I've made a mention of it essentially making the graph 2D. Let me know if you think that is sufficient

**Review by aldebaran30701 — APPROVED** [review]

**Review by Atlas139mkm — APPROVED** [review]

**Review by arcturus2 — APPROVED** [review]

**Pull request merged**